tests/bluetooth/tester/src/btp/btp_ias.h

```diff
@@ -0,0 +1,13 @@
+/* btp_ias.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* events */
+#define IAS_EV_OUT_ALERT_ACTION		0x80
+struct ias_alert_action_ev {
+	uint32_t alert_lvl;
```

Comment:
`uint8_t`? Or should we even just use `enum bt_ias_alert_lvl`?

Reply:
done